### PR TITLE
[RFC] Html Encode and Decode

### PR DESCRIPTION
**Pull request by jcroll**
If you are OK with this concept I could add the functions to the static class as well tests. As well docs.

### CODE REVIEW

File: src/Stringy.php [hunk 1]
```diff
@@ -1461,4 +1461,30 @@ public function regexReplace($pattern, $replacement, $options = 'msr')
 
         return static::create($str, $this->encoding);
     }
+
+    /**
+     * Convert all applicable characters to HTML entities.
+     *
+     * @param  int     $flags See http://php.net/manual/en/function.htmlentities.php
```
Comment (GrahamCampbell) [inline src/Stringy.php]: int|null

File: src/Stringy.php [hunk 2]
```diff
@@ -1461,4 +1461,30 @@ public function regexReplace($pattern, $replacement, $options = 'msr')
...
+    {
+        $str = htmlentities($this->str, $flags, $this->encoding);
+
+        return static::create($str, $this->encoding);
+    }
+
+    /**
+     * Convert all HTML entities to their applicable characters.
+     *
+     * @param  int     $flags See http://php.net/manual/en/function.html-entity-decode.php
```
Comment (GrahamCampbell) [inline src/Stringy.php]: int|null

**Comment by jcroll** [comment]
Ok I wrote tests, methods for the concrete and static classes, and docs.

What you think?

**Comment by danielstjules** [comment]
Looks solid, thanks! :smile:

**Comment by jcroll** [comment]
Cool thanks @danielstjules

**Pull request merged**